lib/Ebook.php

```diff
@@ -2424,10 +2479,22 @@ public static function GetAllByFilter(string $query = null, array $tags = [], En
 		}
 
 		if($query !== null && $query != ''){
-			$query = trim(preg_replace('|[^a-zA-Z0-9 ]|ius', ' ', Formatter::RemoveDiacritics($query)));
-			$query = sprintf('"%s"', $query);  // Require an exact match via double quotes.
-			$whereCondition .= ' and match(e.IndexableText) against(? in boolean mode) ';
+			// Preserve quotes in the query so the user can enter, e.g., "war and peace" for an exact match.
+			$query = trim(preg_replace('|[^a-zA-Z0-9" ]|ius', ' ', Formatter::RemoveDiacritics($query)));
+
+			$relevanceScoreField = ', (
+				match(e.Title) against (?) * ' . EBOOK_SEARCH_WEIGHT_TITLE . ' +
+				match(e.IndexableAuthors) against (?) * ' . EBOOK_SEARCH_WEIGHT_AUTHORS . ' +
+				match(e.IndexableCollections) against (?) * ' . EBOOK_SEARCH_WEIGHT_COLLECTIONS . ' +
+				match(e.IndexableText) against (?)
+			) as relevance_score ';
```

Comment:
Use CamelCase not snake_case for consistency.

Reply:
Done.